### PR TITLE
feat(perf-view): Transaction Summary page

### PR DESCRIPTION
**Pull request by dashed**
**WORK IN PROGRESS**

## TODO

- [ ] graph - @markstory 
- [ ] delete un-used commits 
- [ ] fix formatting for user impact & apdex
- [x] fetching abstraction for `events2` endpoint

### CODE REVIEW

**Comment by dashed** [comment]
@markstory this PR should be good to go. I'm going to revisit it to clean a few things up.

File: src/sentry/static/sentry/app/views/performance/data.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import {Location} from 'history';
 
 import {t} from 'app/locale';
 import {NewQuery} from 'app/types';
+import EventView from 'app/views/eventsV2/eventView';
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/data.tsx]: Maybe we should move this outside of `views`? Could `app/utils/discover/eventView` work?

File: src/sentry/static/sentry/app/views/performance/transaction_summary/breadcrumb.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+    const summaryTarget = transactionSummaryRouteWithEventView({
+      orgSlug: organization.slug,
+      transaction: transactionName,
+      projectID: eventView.project.map(id => String(id)),
+    });
+
+    crumbs.push(
+      <span key="breadcrumb-sep">
+        <StyledIcon src="icon-chevron-right" />
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/transaction_summary/breadcrumb.tsx]: Should we use the new `app/icons/iconChevron`?

File: src/sentry/static/sentry/app/views/performance/transaction_summary/table.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    const {eventView, tableData} = this.props;
+
+    const tableDataMeta = tableData && tableData.meta ? tableData.meta : undefined;
+
+    const columnOrder = eventView.getColumns();
+
+    const generateSortLink = () => undefined;
+
+    return columnOrder.map((column, index) => (
+      <HeaderCell column={column} tableData={tableData} key={index}>
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/transaction_summary/table.tsx]: I would be nice to revisit this and figure out how to consolidate this table code with discover results. I feel like we can get to an API where the table wrapper has options for resizing/sorting/editing that enable a dynamic or static table to be created.

File: src/sentry/static/sentry/app/views/performance/transaction_summary/table.tsx [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  align-items: center;
+  margin: 0 0 ${space(1)} 0;
+`;
+
+export const HeaderButtonContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+
+  /* Hovercard anchor element when features are disabled. */
+  & > span {
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/transaction_summary/table.tsx]: Does this exist in performance views?

**Review by markstory — APPROVED** [review]
Approving so we can unblock other work on this page that isn't accessible by customers yet.

**Pull request merged**